pkg/controllers/management/gke/gke_cluster_handler.go

```diff
@@ -154,9 +154,11 @@ func (e *gkeOperatorController) onClusterChange(key string, cluster *mgmtv3.Clus
 	switch phase {
 	case "creating":
 		// set provisioning to unknown
-		cluster, err = e.SetUnknown(cluster, apimgmtv3.ClusterConditionProvisioned, "")
-		if err != nil {
-			return cluster, err
+		if failureMessage == "" {
+			cluster, err = e.SetUnknown(cluster, apimgmtv3.ClusterConditionProvisioned, "")
+			if err != nil {
+				return cluster, err
+			}
 		}
```

Comment:
Not sure if this is needed because the same call is also made below. @rmweir: What are your thoughts?

Reply:
I'm not sure why it's here except that it's the same in EKS